pennylane/qcut/cutcircuit_mc.py

```diff
@@ -692,22 +691,31 @@ def expand_fragment_tapes_mc(
     for tape in tapes:
         frag_config = []
         for shot in range(shots):
-            with AnnotatedQueue() as q:
-                for op in tape.operations:
-                    w = op.wires[0]
-                    if isinstance(op, PrepareNode):
-                        MC_STATES[prep_settings[op.id][shot]](w)
-                    elif not isinstance(op, MeasureNode):
-                        qml.apply(op)
-
-                for meas in tape.measurements:
-                    qml.apply(meas)
-                for op in tape.operations:
-                    if isinstance(op, MeasureNode):
-                        meas_w = op.wires[0]
+            expanded_circuit_operations = []
+            expanded_circuit_measurements = []
+            for op in tape.operations:
+                w = op.wires[0]
+                if isinstance(op, PrepareNode):
+                    expanded_circuit_operations.extend(MC_STATES[prep_settings[op.id][shot]](w))
+                elif not isinstance(op, MeasureNode):
+                    expanded_circuit_operations.append(op)
+
+            for meas in tape.measurements:
+                expanded_circuit_measurements.append(meas)
+            for op in tape.operations:
+                if isinstance(op, MeasureNode):
```

Comment:
note for style, so you can ignore it: if you initiate `expanded_circuit_measurements` as `tape.measurements.copy()`, then you can put this `if` condition as the final `else` in the first loop over the operations now that we don't need queuing

Reply:
Good catch, I added that :+1: